double-conversion/double-to-string.h

```diff
@@ -104,14 +104,18 @@ class DoubleToStringConverter {
   //   ToPrecision(230.0, 2) -> "230"
   //   ToPrecision(230.0, 2) -> "230."  with EMIT_TRAILING_DECIMAL_POINT.
   //   ToPrecision(230.0, 2) -> "2.3e2" with EMIT_TRAILING_ZERO_AFTER_POINT.
+  //
+  // The exponent width may add up to min_exponent_width by adding prefix '0',
+  // but the exponent will never have more than 5 digits as kMaxExponentLength is 5
```

Comment:
No need to mention `kMaxExponentLength`. It's an internal detail.
Simply:
"When min_exponent_width is used for exponential representations. The converter adds leading '0's, to the exponent until the exponent is at least min_exponent_width digits long.
The min_exponent_width is clamped to 5. As such, the exponent may never have more than 5 digits in total."

Reply:
or something similar.